store/src/java/com/zimbra/cs/account/Provisioning.java

```diff
@@ -1048,6 +1047,14 @@ public abstract Account restoreAccount(String emailAddress, String password,
      */
     public abstract void deleteAccount(String zimbraId) throws ServiceException;
 
+    /**
+     * Checking whether the license which is available in the server is valid or not
+     * @throws ServiceException
+     */
+    public void validateLicense() throws ServiceException{
```

Comment:
```suggestion
    public void validateLicense() throws ServiceException {
```

Reply:
This code is removed now due to change the logic.